### PR TITLE
List: move away from styler

### PR DESCRIPTION
**Pull request by aeschli**

### CODE REVIEW

File: src/vs/workbench/contrib/notebook/browser/view/notebookCellList.ts [hunk 1]
```diff
@@ -1313,11 +1311,11 @@ export class NotebookCellList extends WorkbenchList<CellViewModel> implements ID
...
-			if (styles.listBackground.isOpaque()) {
-				content.push(`.monaco-list${suffix} > div.monaco-scrollable-element > .monaco-list-rows { background: ${styles.listBackground}; }`);
-			} else if (!isMacintosh) { // subpixel AA doesn't exist in macOS
-				console.warn(`List with id '${selectorSuffix}' was styled with a non-opaque background color. This will break sub-pixel antialiasing.`);
-			}
+			//if (styles.listBackground.isOpaque()) {
+			content.push(`.monaco-list${suffix} > div.monaco-scrollable-element > .monaco-list-rows { background: ${styles.listBackground}; }`);
+			//} else if (!isMacintosh) { // subpixel AA doesn't exist in macOS
+			//	console.warn(`List with id '${selectorSuffix}' was styled with a non-opaque background color. This will break sub-pixel antialiasing.`);
+			//}
```
Comment (joaomoreno) [inline src/vs/workbench/contrib/notebook/browser/view/notebookCellList.ts:1318]: Still needed?

File: src/vs/workbench/contrib/notebook/browser/diff/notebookDiffList.ts [hunk 2]
```diff
@@ -367,11 +365,11 @@ export class NotebookTextDiffList extends WorkbenchList<DiffElementViewModelBase
...
-			if (styles.listBackground.isOpaque()) {
-				content.push(`.monaco-list${suffix} > div.monaco-scrollable-element > .monaco-list-rows { background: ${styles.listBackground}; }`);
-			} else if (!isMacintosh) { // subpixel AA doesn't exist in macOS
-				console.warn(`List with id '${selectorSuffix}' was styled with a non-opaque background color. This will break sub-pixel antialiasing.`);
-			}
+			//if (styles.listBackground.isOpaque()) {
+			content.push(`.monaco-list${suffix} > div.monaco-scrollable-element > .monaco-list-rows { background: ${styles.listBackground}; }`);
+			//} else if (!isMacintosh) { // subpixel AA doesn't exist in macOS
+			//	console.warn(`List with id '${selectorSuffix}' was styled with a non-opaque background color. This will break sub-pixel antialiasing.`);
+			//}
```
Comment (joaomoreno) [inline src/vs/workbench/contrib/notebook/browser/diff/notebookDiffList.ts:372]: Still needed?

**Comment by joaomoreno** [comment]
@aeschli Apologies for getting to this so late.

File: src/vs/workbench/contrib/notebook/browser/diff/notebookDiffList.ts (hunk 2 above)
Comment (aeschli) [reply src/vs/workbench/contrib/notebook/browser/diff/notebookDiffList.ts:372]: I'll remove them

**Review by joaomoreno — APPROVED** [review]

**Pull request merged**